examples/python-flask/README.md

```diff
@@ -0,0 +1,46 @@
+# Flask Example
+
+This project demonstrates use of verifiable credentials and presentation  for an
+application.
+
+## Dependencies
+
+- Python 3
+- Pip
+- Python 3 virtual environment
+
+```bash
+$ sudo apt update
+$ sudo apt install -y python3.6 python3-pip python3-virtualenv
```

Comment:
I also had to `apt-get install python3-venv`

Reply:
Added.